app/config/config.yml

```diff
@@ -71,6 +71,7 @@ twig:
         legacy_adhesion:        "https://adherents.en-marche.fr/adherer.html"
         legislatives_platform:  "%env(LEGISLATIVES_PLATFORM)%"
         algolia_blacklist:      "%env(ALGOLIA_BLACKLIST)%"
+        url_help_en_marche:     "https://aide.en-marche.fr"
```

Comment:
This URL should be in parameters.yml no ?
Any thought @Remg ?

Reply:
I hope no, because it's an external service who doesn't provide a staging env

Reply:
ok fine !